gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/impl/FlowServiceImpl.java

```diff
@@ -118,12 +128,33 @@ public String getPlatformFlowSchemaForm(final ExecutionContext executionContext)
         return fileContent;
     }
 
+    private void enrichFlow(io.gravitee.repository.management.model.flow.Flow flow) {
+        String envId = GraviteeContext.getCurrentEnvironment();
+        for (var flowRequest : flow.getRequest()) {
+            if ("shared-policy-group-policy".equals(flowRequest.getPolicy())) {
+                try {
+                    String crossId = new ObjectMapper()
+                        .readValue(flowRequest.getConfiguration(), HashMap.class)
```

Comment:
Just a suggestion, I don't know if it's better.
We could have deserialized using this class `io.gravitee.gateway.handlers.sharedpolicygroup.policy.SharedPolicyGroupPolicyConfiguration` ?